xbmc/pvr/channels/PVRChannelGroup.h

```diff
@@ -556,6 +563,7 @@ namespace PVR
     CEventSource<PVREvent> m_events;
     bool m_bIsSelectedGroup = false; /*!< Whether or not this group is currently selected */
     bool m_bStartGroupChannelNumbersFromOne = false; /*!< true if we start group channel numbers from one when not using backend channel numbers, false otherwise */
+    bool m_bSyncChannelGroups = false; /*!< true if channel groups should be synced with the backend, false otherwise */
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

Reply:
looks like you have overlooked this

Reply:
I did or it looked wrong 😉

I think I was going to see if clang-format could deal with comments better. Must the comment start on the same line as variable declaration for the docs generation to work correctly?

Reply:
Hmmm, I'm not sure we should do this. The output with comments looks really bad:
```
    bool m_bSyncChannelGroups =
        false; /*!< true if channel groups should be synced with the backend, false otherwise */
```

Also note that if the comment is still longer than column limit it will not be modified.

```
    bool m_bSyncChannelGroups =
        false; /*!< true if channel groups should be synced with the backend, false otherwise  and some other text to make this really long. */
```

Reply:
I think we will leave this one case as is for now. We need to figure how to handle member doxy comments and clang-format across PVR first.